jekyll/_cci2/building-docker-images.md

```diff
@@ -150,10 +141,12 @@ CircleCI supports multiple versions of Docker. The following are the available v
 Consult the [Stable releases](https://download.docker.com/linux/static/stable/x86_64/) or [Edge releases](https://download.docker.com/linux/static/edge/x86_64/) for the full list of supported versions.
 --->
 
-**Note:** The `version` key is not currently supported on CircleCI server installations. Contact your system administrator for information about the Docker version installed in your remote Docker environment.
+The `version` key is not currently supported on CircleCI server installations. Contact your system administrator for information about the Docker version installed in your remote Docker environment.
+{: class="alert alert-info"}
 
 ## Separation of environments
```

Comment:
@sebastian-lerner There's a whole section on separation of environments here but I don't the info to be able to know how it needs to be updated. Could you point me in the right direction?

Reply:
@brentmmarks any idea if any of these subsections are still applicable? i imagine the first two arent?

Reply:
think you are right